test/jdk/sun/misc/CopyMemory.java

```diff
@@ -207,6 +207,11 @@ private static Unsafe getUnsafe() throws Exception {
     public static void main(String[] args) throws Exception {
         Unsafe unsafe = getUnsafe();
 
+        // long seed = 6742745864802755133L;
+        long seed = random.nextLong();
+        random.setSeed(seed);
+        System.out.println("Seed set to "+ seed);
+
```

Comment:
Yes, but I believe we should adopt this for the future since reproducing random test failures is extremely difficult without knowing the seed of the RNG.

Reply:
Removed.